pkg/receiver/activedirectoryinvreceiver/factory.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package activedirectoryinvreceiver
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver"
+)
+
+const (
+	// The value of "type" key in configuration.
+	typeStr = "activedirectoryinv"
+)
+
+// NewFactory creates a factory for Active Directory Inventory receiver
+func NewFactory() receiver.Factory {
+	return receiver.NewFactory(
+		typeStr,
+		CreateDefaultConfig,
+		receiver.WithLogs(createLogsReceiver, component.StabilityLevelAlpha),
+	)
+}
+
+// CreateDefaultConfig creates the default configuration for the receiver
+func CreateDefaultConfig() component.Config {
```

Comment:
Are these defaults consistent with the defaults for the [Windows Active Directory Inventory Source](https://help.sumologic.com/docs/send-data/installed-collectors/sources/windows-active-directory-inventory-source/)?

Reply:
After checking again, it looks like the following should be the defaults

- DN = ""
- Attributes = [name, mail, department, manager, memberOf]
- PollInterval = 24 hours

Reply:
The Distinguished name (DN) shouldn't be empty and must have a user domain. So the default value is a placeholder string.

Reply:
will make the default empty and add a note about the format of the DN in the readme